forms-flow-api-utils/src/formsflow_api_utils/utils/pdf.py

```diff
@@ -123,3 +125,42 @@ def save_pdf_local(result, file_name="Pdf.pdf"):
     """Save html content as pdf response."""
     with open(file_name, "wb") as file:
         file.write(result)
+
+
+
+def get_pdf_from_html_string(html_content, chrome_driver_path=None, args=None):
+    """
+    Generate PDF from HTML content string using Chrome/Chromium.
+    
+    Args:
+        html_content (str): HTML content to convert to PDF
+        chrome_driver_path (str): Path to chrome driver
+        args (dict): Dictionary of arguments to pass to Chrome
+        
+    Returns:
+        bytes: PDF content as bytes
+    """
+    try:
+        # Create a temporary file to store the HTML content
+        fd, temp_html_path = tempfile.mkstemp(suffix='.html')
+        with os.fdopen(fd, 'w') as f:
+            f.write(html_content)
+        
+        # Use file:// protocol to access the temporary file
+        temp_url = f"file://{temp_html_path}"
+        
+        # Use the existing function to generate PDF from URL
+        from formsflow_api_utils.utils.pdf import get_pdf_from_html
+        pdf = get_pdf_from_html(temp_url, chrome_driver_path, args)
```

Comment:
Whether we need this import here (`from formsflow_api_utils.utils.pdf import get_pdf_from_html`)? Can we call get_pdf_from_html() directly since it is in the same file?

Reply:
Good catch, removed them